### ISSUE TITLE
rpi-homeport

### ISSUE THREAD

**Issue by vielmetti** [bigeasy/homeport#17]
Make the whole system work on a Raspberry Pi. At minimum this means a separate build that targets ARM systems.

**Comment by bigeasy** [comment]
Does Docker run on RPi?

**Comment by vielmetti** [comment]
It does via Hypriot http://blog.hypriot.com/

http://blog.hypriot.com/getting-started-with-docker-on-your-arm-device/

I've got it running on a Pi, and have a Pi 2 ready to give it another try on.

**Comment by vielmetti** [comment]
In particular I have this building right now

https://github.com/nieleyde/rpi-nodered

which looks it it runs under Arch Linux.

**Comment by vielmetti** [comment]
Thinking about this some more - the package management is probably the biggest issue. Is it reasonable to want to build for something that uses something other than `apt-get` ? e.g. to target `yum` ?

**Comment by bigeasy** [comment]
Yes. You'd start from a different base and use yum instead of apt-get. The first could be a parameter to create, the second could be a default formula, because  we use a "formula" to install from `rpm`, `gem` and `pip` already, we may as well use one for the default action.

So, yes, then you could use Homeport to build from a base other than Ubuntu.
